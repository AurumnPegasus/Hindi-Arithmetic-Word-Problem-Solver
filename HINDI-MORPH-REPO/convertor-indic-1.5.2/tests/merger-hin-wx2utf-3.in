<Sentence id='3'>
1	((	NP	
1.1	((	NP	<fs etype="ne" esubtype="location"  >
1.1.1	BArawa	NNP	<fs af='BArawa,n,m,sg,3,o,0,0'  >
	))
1.2	kA	PSP	<fs af='kA,psp,m,sg,,d,kA,kA'  >
	))
2	((	NP	
2.1	((	NP	<fs etype="ne" esubtype="location"  >
2.1.1	amerikA	NNP	<fs af='amerikA,n,m,sg,3,o,0,0'  >
	))
2.2	ke	PSP	<fs af='kA,psp,m,sg,,o,kA,kA'  >
2.3	pAsa	NST	<fs af='pAsa,adv,,,,,,'  >
	))
3	((	NP	
3.1	((	NP	<fs etype="measx" esubtype="Rupees" value="2000000000.0"  >
3.1.1	((	NP	<fs etype="numx" value="2000000000.0"  >
3.1.1.1	2	QCC	<fs af='2,num,,,,,,' etype="numx" value="2.0"  >
3.1.1.2	sO	QCC	<fs af='sO,adj,any,pl,,d,,' etype="numx" value="100.0"  >
3.1.1.3	karodZa	QC	<fs af='karodZa,adj,any,pl,,d,,' etype="numx" value="10000000.0"  >
	))
3.1.2	rupayoM	NN	<fs af='rupayA,n,m,pl,3,o,0,0' esubtype="Rupees" etype="measx" value="1.0"  >
	))
3.2	kA	PSP	<fs af='kA,psp,m,sg,,d,kA,kA'  >
	))
4	((	NP	
4.1	karja	NN	<fs af='karja,n,m,sg,3,d,0,0'  >
	))
5	((	VGF	
5.1	hE	VM	<fs af='hE,v,any,sg,3,,hE,hE'  >
5.2	.	SYM	<fs af='.,punc,,,,,,'  >
	))
</Sentence>

