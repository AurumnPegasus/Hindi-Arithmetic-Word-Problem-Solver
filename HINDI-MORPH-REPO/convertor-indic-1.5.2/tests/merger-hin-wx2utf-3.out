<Sentence id='3'>			
1	((	NP	
1.1	((	NP	<fs  esubtype="location"  etype="ne">
1.1.1	भारत	NNP	<fs af='भारत,n,m,sg,3,o,0,0'>
	))
1.2	का	PSP	<fs af='का,psp,m,sg,,d,का,kA'>
	))
2	((	NP	
2.1	((	NP	<fs  esubtype="location"  etype="ne">
2.1.1	अमेरिका	NNP	<fs af='अमेरिका,n,m,sg,3,o,0,0'>
	))
2.2	के	PSP	<fs af='का,psp,m,sg,,o,का,kA'>
2.3	पास	NST	<fs af='पास,adv,,,,,,'>
	))
3	((	NP	
3.1	((	NP	<fs  esubtype="Rupees"  etype="measx"  value="2000000000.0">
3.1.1	((	NP	<fs  etype="numx"  value="2000000000.0">
3.1.1.1	2	QCC	<fs af='2,num,,,,,,' etype="numx"  value="2.0">
3.1.1.2	सौ	QCC	<fs af='सौ,adj,any,pl,,d,,' etype="numx"  value="100.0">
3.1.1.3	करोड़	QC	<fs af='करोड़,adj,any,pl,,d,,' etype="numx"  value="10000000.0">
	))
3.1.2	रुपयों	NN	<fs af='रुपया,n,m,pl,3,o,0,0' esubtype="Rupees"  etype="measx"  value="1.0">
	))
3.2	का	PSP	<fs af='का,psp,m,sg,,d,का,kA'>
	))
4	((	NP	
4.1	कर्ज	NN	<fs af='कर्ज,n,m,sg,3,d,0,0'>
	))
5	((	VGF	
5.1	है	VM	<fs af='है,v,any,sg,3,,है,hE'>
5.2	.	SYM	<fs af='.,punc,,,,,,'>
	))
</Sentence>			
			
